tools/sass_bundle.ts

```diff
@@ -0,0 +1,29 @@
+import {dirname, join} from 'path';
+import {readFileSync, writeFileSync, readdirSync, lstatSync} from 'fs';
+
+const Bundler = require('scss-bundle').Bundler;
+const minimist = require('minimist');
+
+/** The root of the bazel workspace (sandbox). */
+const workspaceRoot = process.cwd();
+
+export async function main(args: string[]): Promise<number> {
+  const parsedArgs: {srcs: string, output: string, entry: string} = minimist(args);
+  const inputFiles = parsedArgs.srcs.split(',');
+
+  return new Bundler().Bundle(join(workspaceRoot, parsedArgs.entry), inputFiles).then(result => {
```

Comment:
did you determine if the CLI in scss-bundle is not suitable? if so, comment why in the fileoverview here

Reply:
I couldn't get the `nodejs_binary` to work with the package's existing CLI and it was less work to keep this. It's also easier to debug the calls from bazel this way. Added a comment.